.circleci/config.yml

```diff
@@ -102,10 +102,30 @@ jobs:
           POSTGRES_DB: ruby30
           POSTGRES_PASSWORD: ""
 
+  ruby-31:
+    <<: *default_job
+    steps:
+      - shared_steps
+      # Run the tests against the versions of Rails that support Ruby 3.1
+      - run: bundle exec appraisal install
+      - run: bundle exec appraisal rails61 rspec
+    docker:
+      - image: cimg/ruby:3.1-browsers
```

Comment:
Do we need the browsers image here, or is the standard `ruby:3.1` with the orb enough?

Reply:
If I understand correctly, the `browsers` variant supplies the dependencies needed for the orb to work:

> The browsers variant is designed to work in conjunction with the CircleCI Browser Tools orb. You can use the orb to install a version of Google Chrome and/or Firefox into your build.

https://circleci.com/developer/images/image/cimg/ruby#browsers

But I'll push up a quick test commit just to confirm.

Reply:
Ah, actually I forgot for a bit that this PR only uses `cimg/ruby` in the 3.1 tests.  And there is a difference of 7s (with `-browsers`) vs 23s (without `-browsers) for "Install Google Chrome".  We could try testing each version a few more times to confirm the speed difference.  I've also opened that ticket w/ Circle to confirm.
What do you prefer to do at this point?

Reply:
@nickcharlton I went ahead and did some speed testing and it does seem like the difference is probably that some stuff is preinstalled on `-browsers`, which makes the browser installation faster.  However, I'd bet it probably makes the image download longer also.

What I saw:

| test run | "-browsers"? | env spin up (incl download) | chrome install | total |
| - | - | - | - | - |
| [link](https://app.circleci.com/pipelines/github/rinsed-org/administrate/33/workflows/bca36891-68a7-4603-9275-29eee46fef38/jobs/101) | yes | 30s | 9s | 39s |
| [link](https://app.circleci.com/pipelines/github/rinsed-org/administrate/36/workflows/8857a2fc-5a3f-4ce6-a3f2-648dc71bf1d2/jobs/114) | yes | 6s | 7s | 13s |
| [link](https://app.circleci.com/pipelines/github/rinsed-org/administrate/38/workflows/d18672dd-3987-4c20-9103-7550f7ce0288/jobs/119) | yes | 13s | 7s | 20s |
| [link](https://app.circleci.com/pipelines/github/rinsed-org/administrate/34/workflows/bd77b117-6010-4612-b642-88948c808e39/jobs/105) | no | 5s | 18s | 23s |
| [link](https://app.circleci.com/pipelines/github/rinsed-org/administrate/37/workflows/1b3b74a7-8307-48ff-9774-10fbc8cae36e/jobs/117) | no | 12s | 17s | 29s |
| [link](https://app.circleci.com/pipelines/github/rinsed-org/administrate/39/workflows/6ea45fc7-8e97-45f5-b5c0-7f5bb123a290/jobs/126) | no | 3s | 19s | 22s |

Lmk what you think and which you prefer. Circle has enough variability in timing that it's hard to really get a firm idea of things 😕

Reply:
Oh I see. Right, it is intentional on Circle's part. I didn't think to even look at that, so thanks for highlighting it.

I think we should go with what they recommend — it's a pity your table shows how variable the runs are (thanks for doing that!).

Reply:
👍 sounds good. I've force-pushed to remove the commit that tried out the non-`-browsers` variant.